talkative_task_robot.rb

```diff
@@ -51,7 +13,7 @@
 last_task = "last task"
 to_dos << last_task
 puts(" You just remembered you want to do something when all the other stuff is done #{last_task}")
-user = {}
+user = {name: "Greg"}
 q1 = {question1: " First thing you need to do #{first_task}"}
```

Comment:
You should put these 5 questions into an array. Also, single letter variable names are not very descriptive.